### PR TITLE
Update KB submodule and Release v1.2.16

### PR DESCRIPTION
**Pull request by nmasdoufi-ol**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -13,7 +13,7 @@ license_files = LICENSE
 long_description = file: README.md
 long_description_content_type = text/markdown
 url = https://github.com/Ostorlab/ostorlab
-version=1.2.14
+version=1.2.16
```
Comment (ybadaoui-ostorlab) [inline setup.cfg:16]:
```suggestion
version=1.2.15
```

File: setup.cfg [hunk 2]
```diff
@@ -13,7 +13,7 @@ license_files = LICENSE
 long_description = file: README.md
 long_description_content_type = text/markdown
 url = https://github.com/Ostorlab/ostorlab
-version=1.2.14
+version=1.2.15
```
Comment (deadly-panda) [inline setup.cfg:16]:
```suggestion
version=1.2.16
```

**Review by deadly-panda — APPROVED** [review]

**Comment by PiranhaSa** [comment]
you have conflict

**Comment by nmasdoufi-ol** [comment]
> you have conflict

Resolved.

**Review by deadly-panda — APPROVED** [review]

**Pull request merged**